From efcb9f8e4bbf154491e18d6cf8cbe8dfd4982c6e Mon Sep 17 00:00:00 2001
From: Dylan Yip <dylan.yip@xilinx.com>
Date: Tue, 9 Mar 2021 12:36:35 -0800
Subject: [PATCH 1763/1851] drm/of: Correct documentation for
 drm_of_find_possible_crtcs()

commit 3c08e5fce8c1d0bd2dd769a446e31b5b62427a91 from
https://github.com/Xilinx/linux-xlnx.git

drm_of_find_possible_crtcs() documentation currently states that it
expects an encoder port OF node. However, it really expects the entire
device OF node so correct the documentation and variable naming.

Signed-off-by: Dylan Yip <dylan.yip@xilinx.com>
Signed-off-by: Jianqiang Chen <jianqiang.chen@xilinx.com>
Acked-by: Varunkumar Allagadapa <varunkumar.allagadapa@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/gpu/drm/drm_of.c | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/drivers/gpu/drm/drm_of.c b/drivers/gpu/drm/drm_of.c
index 6becf63f9166..c0be9321cb32 100644
--- a/drivers/gpu/drm/drm_of.c
+++ b/drivers/gpu/drm/drm_of.c
@@ -48,23 +48,23 @@ uint32_t drm_of_crtc_port_mask(struct drm_device *dev,
 EXPORT_SYMBOL(drm_of_crtc_port_mask);
 
 /**
- * drm_of_find_possible_crtcs - find the possible CRTCs for an encoder port
+ * drm_of_find_possible_crtcs - find the possible CRTCs for an encoder
  * @dev: DRM device
- * @port: encoder port to scan for endpoints
+ * @encoder: encoder OF node to scan for endpoints
  *
- * Scan all endpoints attached to a port, locate their attached CRTCs,
+ * Scan all endpoints attached to a encoder, locate their attached CRTCs,
  * and generate the DRM mask of CRTCs which may be attached to this
  * encoder.
  *
  * See Documentation/devicetree/bindings/graph.txt for the bindings.
  */
 uint32_t drm_of_find_possible_crtcs(struct drm_device *dev,
-				    struct device_node *port)
+				    struct device_node *encoder)
 {
 	struct device_node *remote_port, *ep;
 	uint32_t possible_crtcs = 0;
 
-	for_each_endpoint_of_node(port, ep) {
+	for_each_endpoint_of_node(encoder, ep) {
 		remote_port = of_graph_get_remote_port(ep);
 		if (!remote_port) {
 			of_node_put(ep);
-- 
2.31.1

